example/react-rest-hatchify-backend/.eslintrc.cjs

```diff
@@ -0,0 +1,14 @@
+module.exports = {
```

Comment:
i added a new example package that we can use to manually validate react-rest against a hatchify backend with multiple lists, delete, edit, create.

will do e2e tests against it in a future task